modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/APMAgentSettings.java

```diff
@@ -238,20 +240,41 @@ public void setAgentSetting(String key, String value) {
         }
     );
 
-    public static final Setting<List<String>> APM_TRACING_NAMES_INCLUDE_SETTING = Setting.stringListSetting(
-        APM_SETTING_PREFIX + "names.include",
+    // TODO eventually remove deprecated setting
+    public static final Setting<List<String>> TRACING_APM_NAMES_INCLUDE_SETTING = Setting.stringListSetting(
```

Comment:
These settings are not APMAgent specific, should we rename `APMAgentSettings` to `APMSettings` or `TelemetrySettings` in a follow up PR?